docs/query-metadata-style-guide.md

```diff
@@ -157,6 +157,12 @@ If your query is a security query, use one or more `@tags` to associate it with
 
 When you tag a query like this, the associated CWE pages from [MITRE.org](http://cwe.mitre.org/index.html) will automatically appear in the reference section of its associated qhelp file.
 
+#### Metric/summary `@tags`
+
+Queries with the `summary` tag represent metrics or other pieces of summary information. If SARIF output is used, the results of these queries can be found at `run[].properties.metricResults`.
```

Comment:
```suggestion
Queries with `@kind metric` and the `summary` tag represent metrics or other pieces of summary information. If SARIF output is used, the results of these queries can be found at `run[].properties.metricResults`.
```

Reply:
Perhaps also add an entry to the `kind` and `tags` section above?

Reply:
Just looking at this. Should metric queries _always_ have the `summary` tag?

Reply:
More general metric queries without the `summary` tag are permitted by the engine, but not populated into `metricResults`. (My mental model is that "summary metric" queries are a subset of metric queries.)

Reply:
Great. I've ended up moving your `select` suggestion under `QL area`, so the description is split across the doc as with alert queries. Perhaps if metric queries become more widespread we'll want two completely separate top level headings but this works for now IMO.